examples/gallery/images/track_sampling.py

```diff
@@ -25,18 +24,40 @@
 track = pygmt.grdtrack(points=points, grid=grid, newcolname="bathymetry")
 
 fig = pygmt.Figure()
-# Plot the earth relief grid on Cylindrical Stereographic projection, masking
-# land areas
+# Create global map using a Cylindrical Stereographic projection with a
+# width of 15 centimeters
 fig.basemap(region="g", projection="Cyl_stere/150/-20/15c", frame=True)
-fig.grdimage(grid=grid, cmap="gray")
-fig.coast(land="#666666")
-# Plot the sampled bathymetry points using circles (c) of 0.15 cm size
-# Points are colored using elevation values (normalized for visual purposes)
+
+# Set up a colormap for the elevation values of the Earth relief grid
+pygmt.makecpt(cmap="gray", series=[int(grid.min()), int(grid.max()), 10])
```

Comment:
Would [`grd2cpt`](https://www.pygmt.org/v0.7.0/api/generated/pygmt.grd2cpt.html) be better here instead of `makecpt`? We actually don't have an example for `grd2cpt` yet :slightly_smiling_face:

Reply:
Thanks for this suggestions! Can have a look at it.

Reply:
I feel I can not finish the work on changing this example regarding this comment today to get this PR into the v0.8.0 release.